PID_control:
  ros__parameters:
    kp: 1.0
    ki: 0.0
    kd: 0.0
    dt: 0.1
    saturation: 0.0
    standard_pwm: 1550.0

    dead_max: 1530.0
    dead_min: 1470.0

    navigation_topic: "/navigation_data"
    guidance_topic: "/guidance_data"
    rc_topic: "/rc_data"
    control_topic: "/control_data"
